.pipelines/ci/templates/build-powertoys-steps.yml

```diff
@@ -87,6 +87,7 @@ steps:
       **\Microsoft.Plugin.Uri.UnitTests.dll
       **\Wox.Test.dll
       **\*Microsoft.PowerToys.Settings.UI.UnitTests.dll
+      **\UnitTest-ColorPickerUI.dll
```

Comment:
we really need to standardize the naming

Reply:
That would also fix this issue of having to adding projects to CI each time. Something like **\*UnitTests*.dll would be sufficient to add all of them

Reply:
exactly